10_1_graph.js

```diff
@@ -0,0 +1,15 @@
+export function buildGraph(edges) {
+    let graph = Object.create(null);
+    function addEdge(from, to) {
```

Comment:
extract this function outside of buildGraph an add graph as a parameter (graph, from, to). Then return the modified graph as a result. This way you create a pure function that you can also reuse outside of buildGraph.